<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подпись DSA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="eMessageSender">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="eMessageGetter">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>940</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>P=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>G=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eG">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>940</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Переслать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eR">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>S=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eS">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCheck">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>320</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Проверить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eY">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>430</y>
      <width>941</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>430</y>
      <width>29</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eQ">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Q=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>m=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>m=</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblValid">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>320</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalculateY">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>149</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать Y из X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправитель</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Получатель</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
